### PR TITLE
[IMP] thousands separator in Excel export

### LINKED ISSUE #190: Display thousands separator in XLS export
Investigate which Excel number format could be use to display thousands separator in XLS export. This should be done in [to_xlsx_style](https://github.com/OCA/mis-builder/blob/8a70a2607dce54d608e83ecb8f68c4f4f9094219/mis_builder/models/mis_report_style.py#L254).

### PR DESCRIPTION
**Pull request by sbidoul**
closes #190

### CODE REVIEW

File: mis_builder/readme/newsfragments/190.feature [hunk 1]
```diff
@@ -0,0 +1,3 @@
+In Excel export, use a number format with thousands separator. The
+specific separator used depends on the Excel configuraiton (eg regional
```
Comment (pedrobaeza) [inline mis_builder/readme/newsfragments/190.feature]: s/configuraiton/configuration

Comment (sbidoul) [reply mis_builder/readme/newsfragments/190.feature]: @pedrobaeza thanks, fixed.

**Review by pedrobaeza — APPROVED** [review]

**Comment by sbidoul** [comment]
testing again

/ocabot merge

**Comment by pedrobaeza** [comment]
@sbidoul did you explore the possibility of marking the pull request somehow as merged? Can I help in anything?

**Pull request closed without merging**